docs/source/customize.rst

```diff
@@ -32,6 +32,47 @@ HTML templates have their own versioning system and warnings will be triggered w
     This example is using new ``pydoctor`` option, ``--theme=base``. 
     This means that bootstrap CSS will not be copied to build directory.
 
+.. _customize-privacy:
+
+Override objects privacy (show/hide)
+------------------------------------
+
+Pydoctor supports 3 types of privacy:
+
+- ``PRIVATE`` (If object name starts with an underscore and it's not a dunder method): 
+  Rendered in HTML, but hidden via CSS by default.
+
+- ``PUBLIC`` (Everything else):
+  Always rendered and visible in HTML.
+
+- ``HIDDEN`` (Nothing by default):
+  Not rendered at all and no links can be created to hidden objects. 
+  Not present in the search index nor the intersphinx inventory.
+  Basically excluded from API documentation. If a module/package/class is hidden, then all it's members are hidden as well.
+
+When the default rules regarding privacy doesn't fit the use case, you can use the following repeatable option::
```

Comment:
Minor suggestion 

```suggestion
When the default rules regarding privacy doesn't fit your use case,
use the `privacy` command line option.
It can be used multiple times to define multiple privacy rules::
```